src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+    display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-left: 4px;
+    background: url(../images/star-active.svg) center no-repeat;
+  }
+
+  &__star:first-child {
+    margin: 0;
+  }
+
+  &__star:last-child {
+    background: url(../images/star.svg) center no-repeat;
+  }
```

Comment:
It's not about `last-child`, if the modifier is `stars--5` - all stars should be active, if `stars--3` only 3, etc.
But in your case it doesn't work, you have modifier `stars--5`, but there are only 4 active stars 🤷